Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -88,6 +88,8 @@
 #include "VirtualAuthenticatorManager.h"
 #endif // ENABLE(WEB_AUTHN)
 
+using namespace WebCore;
```

Comment:
I thought we stopped doing this in new code because we bundle files together when building now and this often leads to conflicts?

Reply:
Ooh, yes. please don't do this!